src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -710,6 +713,29 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
             }
 
             return UpdateExpression(builder, node.Update(left, right, node.IsRef, node.Type));
+
+            BoundExpression spillReceiver(BoundFieldAccess field, ref BoundSpillSequenceBuilder leftBuilder)
```

Comment:
>BoundFieldAccess field [](start = 42, length = 22)

This feels very confusing. Consider changing this parameter to be the receiver. We get back what we pas in. It looks like we don't need any information from the BoundFieldAccess that we don't have in the receiver.

Reply:
> This feels very confusing. Consider changing this parameter to be the receiver. We get back what we pass in. It looks like we don't need any information from the BoundFieldAccess that we don't have in the receiver.

The name of the method still feels confusing, even with the recent changes.

---
In reply to: [398918937](https://github.com/dotnet/roslyn/pull/42807#discussion_r398918937) [](ancestors = 398918937)